openrtb_ext/imp_connectad.go

```diff
@@ -0,0 +1,7 @@
+package openrtb_ext
+
+type ExtImpConnectAd struct {
+	NetworkId int     `json:"networkId"`
```

Comment:
Per Go naming convention, abbreviations and acronyms should be upper case.
NetworkId -> NetworkID
SiteId -> SiteID